src/styles/blocks.scss/watch.scss

```diff
@@ -0,0 +1,62 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
```

Comment:
your watch.scss is file with all styles for BEM block you can't use positioning for it. So your <div class='container'> is your BEM block because __border, __center e.t.c. are it's elements. If you want to use positioning then wrap your watch into some other block.

or you can use this .container like your wrap container, but then don't bound your watch block with it